app/models/tweeter.rb

```diff
@@ -14,13 +14,12 @@ def self.tweet_new_product(idea, product)
   end
 
   def product_participants(product)
-    if product.user.twitter_nickname
-      participants = [product.user.twitter_nickname]
-    else
-      participants = ["asm"]
+    partners_with_twitter = TransactionLogEntry.product_partners(product.id).order('sum(cents) desc').where.not(twitter_nickname: nil)
```

Comment:
Line is too long. [134/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.